### PR TITLE
Add consistency with phrases "meantime" and "mean time"

### PR DESCRIPTION
**Pull request by danielhuang**
"mean time" is used in a few places while "meantime" is used everywhere else; this would make usage consistent throughout the codebase.

### CODE REVIEW

File: compiler/rustc_lint/src/opaque_hidden_inferred_bound.rs [hunk 1]
```diff
@@ -83,7 +83,9 @@ impl<'tcx> LateLintPass<'tcx> for OpaqueHiddenInferredBound {
                 };
                 // Only check types, since those are the only things that may
                 // have opaques in them anyways.
-                let Some(proj_term) = proj.term.ty() else { return };
+                let Some(proj_term) = proj.term.ty() else {
+                    return;
+                };
```
Comment (compiler-errors) [inline compiler/rustc_lint/src/opaque_hidden_inferred_bound.rs:88]: Why did you change this?

File: compiler/rustc_lint/src/opaque_hidden_inferred_bound.rs [hunk 2]
```diff
@@ -114,7 +116,9 @@ impl<'tcx> LateLintPass<'tcx> for OpaqueHiddenInferredBound {
                 // type in our opaque type.
                 let proj_replacer = &mut BottomUpFolder {
                     tcx: cx.tcx,
-                    ty_op: |ty| if ty == proj_ty { proj_term } else { ty },
+                    ty_op: |ty| {
+                        if ty == proj_ty { proj_term } else { ty }
+                    },
```
Comment (workingjubilee) [inline compiler/rustc_lint/src/opaque_hidden_inferred_bound.rs:121]: this doesn't seem related.

**Review by davidtwco — APPROVED** [review]
r=me after you revert the unrelated code format changes

**Comment by gl-security** [comment]
Can you actually contribute to something more useful? this is just larping to make your chart good for an interview ?

**Comment by danielhuang** [comment]
Removed unnecessary changes; they were put there by rustfmt.
